### ISSUE TITLE
Security issue on lodash

### ISSUE THREAD

**Issue by jankapunkt** [aldeed/js-message-box#11]
NPM audit reports a high severity security issue on lodash, which this package depends on:

```
│ High          │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ simpl-schema                                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ simpl-schema > message-box > lodash                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1065                       
```

also the question, whether it is really needed to include the whole lodash package? This makes `simpl-schema` grow unnecessarily IMO.

**Comment by jankapunkt** [comment]
PR opened at #12

**Comment by aldeed** [comment]
Thanks, published in 0.2.2.

Regarding lodash, this package previously depended on the individual lodash packages, but they had security issues and were no longer supported. Webpack should tree shake the `lodash` package if properly configured, so hopefully only the parts of lodash that are used are getting included.

That said, I think the two parts of lodash that are used by this package could be implemented using different packages or no package at all, so I'll make a separate issue for that.
